e2e/utils.go

```diff
@@ -134,6 +156,22 @@ func ctsSetup(t *testing.T, srv *testutil.TestServer, tempDir string, taskConfig
 
 func ctsSetupTLS(t *testing.T, srv *testutil.TestServer, tempDir string, taskConfig string, tlsConfig tlsConfig) *api.Client {
 	cleanup := testutils.MakeTempDir(t, tempDir)
+
+	// Setup CA path directory if provided
+	var cleanupCAPath func() error
+	if len(tlsConfig.caPath) != 0 {
+		// Create CA Path directory
+		cleanupCAPath = testutils.MakeTempDir(t, tlsConfig.caPath)
+
+		// Get all test certs and move them to the CA path directory
+		certs := testutils.FindFileMatches(t, testCertPath, "*cert*.pem")
```

Comment:
Should we be moving all test certificates? If we're more selective, we can then test that cert A and B in the CA path are valid and cert C that is not in the directory does not work.